git_machete/cmd.py

```diff
@@ -1979,7 +1999,12 @@ def parse_direction(b, down_pick_mode):
             read_definition_file()
             slide_out(params or [current_branch()])
         elif cmd in ("s", "status"):
-            expect_no_param(parse_options(args, "l", ["list-commits"]))
+            expect_no_param(parse_options(args, "l", ["list-commits", "color="]))
+            if opt_color not in ("always", "auto", "never"):
```

Comment:
Is it now consistent with the help for `status` which says that value for `--color` is optional?